// mathjs/lib/entry/dependenciesNumber/dependenciesStirlingS2.generated.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.stirlingS2Dependencies = void 0;

var _dependenciesAddScalar = require("./dependenciesAddScalar.generated");

var _dependenciesCombinations = require("./dependenciesCombinations.generated");

var _dependenciesDivideScalar = require("./dependenciesDivideScalar.generated");

var _dependenciesFactorial = require("./dependenciesFactorial.generated");

var _dependenciesIsInteger = require("./dependenciesIsInteger.generated");

var _dependenciesIsNegative = require("./dependenciesIsNegative.generated");

var _dependenciesLarger = require("./dependenciesLarger.generated");

var _dependenciesMultiplyScalar = require("./dependenciesMultiplyScalar.generated");

var _dependenciesPow = require("./dependenciesPow.generated");

var _dependenciesSubtract = require("./dependenciesSubtract.generated");

var _dependenciesTyped = require("./dependenciesTyped.generated");

var _factoriesNumber = require("../../factoriesNumber.js");

/**
 * THIS FILE IS AUTO-GENERATED
 * DON'T MAKE CHANGES HERE
 */
var stirlingS2Dependencies = {
  addScalarDependencies: _dependenciesAddScalar.addScalarDependencies,
  combinationsDependencies: _dependenciesCombinations.combinationsDependencies,
  divideScalarDependencies: _dependenciesDivideScalar.divideScalarDependencies,
  factorialDependencies: _dependenciesFactorial.factorialDependencies,
  isIntegerDependencies: _dependenciesIsInteger.isIntegerDependencies,
  isNegativeDependencies: _dependenciesIsNegative.isNegativeDependencies,
  largerDependencies: _dependenciesLarger.largerDependencies,
  multiplyScalarDependencies: _dependenciesMultiplyScalar.multiplyScalarDependencies,
  powDependencies: _dependenciesPow.powDependencies,
  subtractDependencies: _dependenciesSubtract.subtractDependencies,
  typedDependencies: _dependenciesTyped.typedDependencies,
  createStirlingS2: _factoriesNumber.createStirlingS2
};
exports.stirlingS2Dependencies = stirlingS2Dependencies;
