// mathjs/lib/entry/dependenciesAny/dependenciesLusolve.generated.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.lusolveDependencies = void 0;

var _dependenciesDenseMatrixClass = require("./dependenciesDenseMatrixClass.generated");

var _dependenciesLsolve = require("./dependenciesLsolve.generated");

var _dependenciesLup = require("./dependenciesLup.generated");

var _dependenciesMatrix = require("./dependenciesMatrix.generated");

var _dependenciesSlu = require("./dependenciesSlu.generated");

var _dependenciesTyped = require("./dependenciesTyped.generated");

var _dependenciesUsolve = require("./dependenciesUsolve.generated");

var _factoriesAny = require("../../factoriesAny.js");

/**
 * THIS FILE IS AUTO-GENERATED
 * DON'T MAKE CHANGES HERE
 */
var lusolveDependencies = {
  DenseMatrixDependencies: _dependenciesDenseMatrixClass.DenseMatrixDependencies,
  lsolveDependencies: _dependenciesLsolve.lsolveDependencies,
  lupDependencies: _dependenciesLup.lupDependencies,
  matrixDependencies: _dependenciesMatrix.matrixDependencies,
  sluDependencies: _dependenciesSlu.sluDependencies,
  typedDependencies: _dependenciesTyped.typedDependencies,
  usolveDependencies: _dependenciesUsolve.usolveDependencies,
  createLusolve: _factoriesAny.createLusolve
};
exports.lusolveDependencies = lusolveDependencies;
