// mathjs/lib/entry/dependenciesAny/dependenciesSqrtm.generated.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.sqrtmDependencies = void 0;

var _dependenciesAbs = require("./dependenciesAbs.generated");

var _dependenciesAdd = require("./dependenciesAdd.generated");

var _dependenciesIdentity = require("./dependenciesIdentity.generated");

var _dependenciesInv = require("./dependenciesInv.generated");

var _dependenciesMax = require("./dependenciesMax.generated");

var _dependenciesMultiply = require("./dependenciesMultiply.generated");

var _dependenciesSize = require("./dependenciesSize.generated");

var _dependenciesSqrt = require("./dependenciesSqrt.generated");

var _dependenciesSubtract = require("./dependenciesSubtract.generated");

var _dependenciesTyped = require("./dependenciesTyped.generated");

var _factoriesAny = require("../../factoriesAny.js");

/**
 * THIS FILE IS AUTO-GENERATED
 * DON'T MAKE CHANGES HERE
 */
var sqrtmDependencies = {
  absDependencies: _dependenciesAbs.absDependencies,
  addDependencies: _dependenciesAdd.addDependencies,
  identityDependencies: _dependenciesIdentity.identityDependencies,
  invDependencies: _dependenciesInv.invDependencies,
  maxDependencies: _dependenciesMax.maxDependencies,
  multiplyDependencies: _dependenciesMultiply.multiplyDependencies,
  sizeDependencies: _dependenciesSize.sizeDependencies,
  sqrtDependencies: _dependenciesSqrt.sqrtDependencies,
  subtractDependencies: _dependenciesSubtract.subtractDependencies,
  typedDependencies: _dependenciesTyped.typedDependencies,
  createSqrtm: _factoriesAny.createSqrtm
};
exports.sqrtmDependencies = sqrtmDependencies;
