// mathjs/lib/entry/dependenciesAny/dependenciesUnitClass.generated.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.UnitDependencies = void 0;

var _dependenciesBigNumberClass = require("./dependenciesBigNumberClass.generated");

var _dependenciesComplexClass = require("./dependenciesComplexClass.generated");

var _dependenciesFractionClass = require("./dependenciesFractionClass.generated");

var _dependenciesAbs = require("./dependenciesAbs.generated");

var _dependenciesAddScalar = require("./dependenciesAddScalar.generated");

var _dependenciesDivideScalar = require("./dependenciesDivideScalar.generated");

var _dependenciesEqual = require("./dependenciesEqual.generated");

var _dependenciesFix = require("./dependenciesFix.generated");

var _dependenciesFormat = require("./dependenciesFormat.generated");

var _dependenciesIsNumeric = require("./dependenciesIsNumeric.generated");

var _dependenciesMultiplyScalar = require("./dependenciesMultiplyScalar.generated");

var _dependenciesNumber = require("./dependenciesNumber.generated");

var _dependenciesPow = require("./dependenciesPow.generated");

var _dependenciesRound = require("./dependenciesRound.generated");

var _dependenciesSubtract = require("./dependenciesSubtract.generated");

var _factoriesAny = require("../../factoriesAny.js");

/**
 * THIS FILE IS AUTO-GENERATED
 * DON'T MAKE CHANGES HERE
 */
var UnitDependencies = {
  BigNumberDependencies: _dependenciesBigNumberClass.BigNumberDependencies,
  ComplexDependencies: _dependenciesComplexClass.ComplexDependencies,
  FractionDependencies: _dependenciesFractionClass.FractionDependencies,
  absDependencies: _dependenciesAbs.absDependencies,
  addScalarDependencies: _dependenciesAddScalar.addScalarDependencies,
  divideScalarDependencies: _dependenciesDivideScalar.divideScalarDependencies,
  equalDependencies: _dependenciesEqual.equalDependencies,
  fixDependencies: _dependenciesFix.fixDependencies,
  formatDependencies: _dependenciesFormat.formatDependencies,
  isNumericDependencies: _dependenciesIsNumeric.isNumericDependencies,
  multiplyScalarDependencies: _dependenciesMultiplyScalar.multiplyScalarDependencies,
  numberDependencies: _dependenciesNumber.numberDependencies,
  powDependencies: _dependenciesPow.powDependencies,
  roundDependencies: _dependenciesRound.roundDependencies,
  subtractDependencies: _dependenciesSubtract.subtractDependencies,
  createUnitClass: _factoriesAny.createUnitClass
};
exports.UnitDependencies = UnitDependencies;
