// mathjs/lib/entry/dependenciesAny/dependenciesMultinomial.generated.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.multinomialDependencies = void 0;

var _dependenciesAdd = require("./dependenciesAdd.generated");

var _dependenciesDivide = require("./dependenciesDivide.generated");

var _dependenciesFactorial = require("./dependenciesFactorial.generated");

var _dependenciesIsInteger = require("./dependenciesIsInteger.generated");

var _dependenciesIsPositive = require("./dependenciesIsPositive.generated");

var _dependenciesMultiply = require("./dependenciesMultiply.generated");

var _dependenciesTyped = require("./dependenciesTyped.generated");

var _factoriesAny = require("../../factoriesAny.js");

/**
 * THIS FILE IS AUTO-GENERATED
 * DON'T MAKE CHANGES HERE
 */
var multinomialDependencies = {
  addDependencies: _dependenciesAdd.addDependencies,
  divideDependencies: _dependenciesDivide.divideDependencies,
  factorialDependencies: _dependenciesFactorial.factorialDependencies,
  isIntegerDependencies: _dependenciesIsInteger.isIntegerDependencies,
  isPositiveDependencies: _dependenciesIsPositive.isPositiveDependencies,
  multiplyDependencies: _dependenciesMultiply.multiplyDependencies,
  typedDependencies: _dependenciesTyped.typedDependencies,
  createMultinomial: _factoriesAny.createMultinomial
};
exports.multinomialDependencies = multinomialDependencies;
