// mathjs/lib/entry/dependenciesAny/dependenciesEigs.generated.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.eigsDependencies = void 0;

var _dependenciesAbs = require("./dependenciesAbs.generated");

var _dependenciesAdd = require("./dependenciesAdd.generated");

var _dependenciesAddScalar = require("./dependenciesAddScalar.generated");

var _dependenciesAtan = require("./dependenciesAtan.generated");

var _dependenciesBignumber = require("./dependenciesBignumber.generated");

var _dependenciesCos = require("./dependenciesCos.generated");

var _dependenciesEqual = require("./dependenciesEqual.generated");

var _dependenciesInv = require("./dependenciesInv.generated");

var _dependenciesMatrix = require("./dependenciesMatrix.generated");

var _dependenciesMultiply = require("./dependenciesMultiply.generated");

var _dependenciesMultiplyScalar = require("./dependenciesMultiplyScalar.generated");

var _dependenciesSin = require("./dependenciesSin.generated");

var _dependenciesSubtract = require("./dependenciesSubtract.generated");

var _dependenciesTyped = require("./dependenciesTyped.generated");

var _factoriesAny = require("../../factoriesAny.js");

/**
 * THIS FILE IS AUTO-GENERATED
 * DON'T MAKE CHANGES HERE
 */
var eigsDependencies = {
  absDependencies: _dependenciesAbs.absDependencies,
  addDependencies: _dependenciesAdd.addDependencies,
  addScalarDependencies: _dependenciesAddScalar.addScalarDependencies,
  atanDependencies: _dependenciesAtan.atanDependencies,
  bignumberDependencies: _dependenciesBignumber.bignumberDependencies,
  cosDependencies: _dependenciesCos.cosDependencies,
  equalDependencies: _dependenciesEqual.equalDependencies,
  invDependencies: _dependenciesInv.invDependencies,
  matrixDependencies: _dependenciesMatrix.matrixDependencies,
  multiplyDependencies: _dependenciesMultiply.multiplyDependencies,
  multiplyScalarDependencies: _dependenciesMultiplyScalar.multiplyScalarDependencies,
  sinDependencies: _dependenciesSin.sinDependencies,
  subtractDependencies: _dependenciesSubtract.subtractDependencies,
  typedDependencies: _dependenciesTyped.typedDependencies,
  createEigs: _factoriesAny.createEigs
};
exports.eigsDependencies = eigsDependencies;
